components/cluster/pcs/files/pcsd.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0'?>
+<!DOCTYPE service_bundle SYSTEM '/usr/share/lib/xml/dtd/service_bundle.dtd.1'>
+<service_bundle type='manifest' name='PCSD_WebGUI'>
+  <service name='application/pcsd' type='service' version='1'>
+      <create_default_instance enabled='false'/>
+      <single_instance/>
+      <dependency
+         name='multi-user-server'
+         grouping='optional_all'
+         type='service'
+         restart_on='none'>
+         <service_fmri value='svc:/milestone/multi-user-server' />
+      </dependency>
+      <dependency name='network' grouping='require_all' restart_on='error' type='service'>
+        <service_fmri value='svc:/milestone/network:default'/>
+      </dependency>
+      <dependency name='filesystem-local' grouping='require_all' restart_on='none' type='service'>
+        <service_fmri value='svc:/system/filesystem/local:default'/>
+      </dependency>
+      <dependency name='autofs' grouping='optional_all' restart_on='error' type='service'>
+        <service_fmri value='svc:/system/filesystem/autofs:default'/>
+      </dependency>
+      <method_context>
+        <method_credential user="root" />
+      </method_context>
+      <exec_method name='start' type='method' exec='/lib/svc/method/svc-pcsd start' timeout_seconds='60'/>
+      <exec_method name='stop'  type='method' exec='/lib/svc/method/svc-pcsd stop' timeout_seconds='60'/>
+      <property_group name='startd' type='framework'>
+         <propval name='ignore_error' type='astring' value='core,signal'/>
+      </property_group>
+      <stability value='Evolving'/>
+      <template>
+         <common_name>
+           <loctext xml:lang='C'>PCSD WebGUI for Corosync/Pacemaker</loctext>
+         </common_name>
+         <documentation>
+           <doc_link name='pcsd' uri='http://www.clusterlabs.org'/>
+           <manpage title='pcsd' section='1M' manpath='/usr/share/man/man8' />
```

Comment:
I asked it already but I missed the answer.
I copied the transform for the move to man.1m from sg3_utils. 
Is this really necessary? Should .8 not be used or it is reserved for a specual purpose? It would be simpler to let the man files .8.